### ISSUE TITLE
Game Crash At Start (White house Heist)

### ISSUE THREAD

**Issue by Jacky10X** [Crackdown-PD2/BigLobby3#22]
`Application has crashed: C++ exception
[string "lib/units/vehicles/vehicledrivingext.lua"]:1604: attempt to index local 'seat' (a nil value)

SCRIPT STACK

on_executed() lib/managers/mission/elementvehicleboarding.lua:70
execute_on_executed() core/lib/managers/mission/coremissionscriptelement.lua:209
_trigger_execute_on_executed() core/lib/managers/mission/coremissionscriptelement.lua:147
on_executed() core/lib/managers/mission/coremissionscriptelement.lua:131
on_executed() lib/managers/mission/missionscriptelement.lua:18
on_executed() core/lib/managers/mission/coreelementrandom.lua:39
execute_on_executed() core/lib/managers/mission/coremissionscriptelement.lua:209
_trigger_execute_on_executed() core/lib/managers/mission/coremissionscriptelement.lua:147
on_executed() core/lib/managers/mission/coremissionscriptelement.lua:131
on_executed() lib/managers/mission/missionscriptelement.lua:18
on_executed() core/lib/managers/mission/coreelementarea.lua:121
_check_amount() core/lib/managers/mission/coreelementarea.lua:270
f() core/lib/managers/mission/coreelementarea.lua:235
update() core/lib/utils/coreevent.lua:158
update() core/lib/managers/mission/coremissionmanager.lua:529
update() core/lib/managers/mission/coremissionmanager.lua:160
core/lib/setups/coresetup.lua:503`

The mainly problem is the helicopter only can contain total 4 players(or Bots) and biglobby have more than that. that is why it cause the crash at start

**Comment by Shad0wlife** [comment]
Oh I remember that.
I'm still waiting for znix' luajit decompiler to check if the mod is even still fully compatible, since there currently is no source dump to use as far as I am aware.

The problem with birth of sky is a bit tricky since it's not exactly a script thing, but a map trigger thing.

Now it IS entirely possible to recognize the current map via script, and if anyone would find a way to RELIABLY register that the mission progressed to the ground, one could probably just teleport the bots down anyways (if their AI doesn't break). My intuitive idea would be that the session host would check for discarding the parachute, but if he forgets that, then there's a problem.

**Comment by Jacky10X** [comment]
@Shad0wlife yes it can work on current payday but new heist like white house is unplayable as far as i know and the birth of the sky problem( the is known as problem). Also, the crew management also is a problem(but it is not critical)

**Comment by Shad0wlife** [comment]
Well, at least part of the problem is that there's a table with the 4 seats of the blackhawk, and the script calculates where to put bots and players based on player count and some stuff - causing it to access indexes of the table that are obviously not filled.

Easiest way to just have it not crash would probably be just trying to put multiple people in the same seats by putting a mod 4 in the calculation of the seat - if that's all of it. Currently don't have the time to actually make it and try it, but if anyone wants to attempt it - go ahead.

**Comment by Jacky10X** [comment]
Can I ask if these problems got fixed? because the white house also got a "secret" but I don't know if the mod also supports the secret or not?

**Comment by Shad0wlife** [comment]
Hmm. I think the code just currently specifically disallows biglobby on the White House map. Maybe that's why it breaks for the 5th player.

**Comment by WelcomeToTheCoomCorner** [comment]
As anyone tried to change numbers in payday 2 files?  it just looks like it checks for 4 players and 3 AI. I did infact go and look at what the _elementareatrigger.lua file and the _elementfilter.lua file as well
